### ISSUE TITLE
Current Folder Bar (a.k.a Navigation bar) only displayed on first folder tab

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#496]
The current folder bar is missing on all additional Thunderbird tabs that show folders. It is only displayed on the very first tab and on tabs with single emails.

Expected behavior: if displayed, current folder toolbar should show in all tabs that show folder contents.

**Comment by crablouis** [comment]
Hi Axel. I'm not seeing this bug behavior. 
QF 6.7 on Thunderbird 115.14.0 (32 bit) 
Win 10 Home

I'm seeing the Nav bar in all folders, whether accessed by selecting a QF tab, or by selecting a folder in the Folder Pane

I saw the github bug notification email, and figured i should say something.

**Comment by RealRaven2000** [comment]
> I'm seeing the Nav bar in all folders, whether accessed by selecting a QF tab, or by selecting a folder in the Folder Pane
> 
> I saw the github bug notification email, and figured i should say something.

thanks that's helpful. I am on Thunderbird 128.1.1 esr so I guess it is a regression from the new version.

I would test with a fresh profile in 115 but Thunderbird updates my profiles to 128 very  aggressively so it;s hard to keep testing in a manner that is backwards compatible. (downgrades are tricky)

**Comment by crablouis** [comment]
Huh, this is the second time this week that I"ve read someone saying their TB is on the ESR channel, and they're at some version way past mine, 115.14.0. 

My TB installation here says it's on the ESR channel as well. 

OS difference?

**Comment by RealRaven2000** [comment]
> Huh, this is the second time this week that I"ve read someone saying their TB is on the ESR channel, and they're at some version way past mine, 115.14.0.
> 
> My TB installation here says it's on the ESR channel as well.
> 
> OS difference?

not really, there are actually 2 ESR channels, one for 115 and one for 128. The new one is not yet rolled out automatically - they usually only do partial roll outs for testing starting with 128.0 and 128.1, so it is a bit of "luck" whether you are on 115 or 128, but you can force update to 128 by opening Help >> about Thunderbird.

The stats say that about 11.6% of users are on 128.1 and 78.7% on 115.* currently:
https://stats.thunderbird.net/#version

**Comment by crablouis** [comment]
Two ESR channels. Interesting.

I'm not seeing any UI for bumping the auto-update in TB>Help>About.  Just the usual ~"yer up to date" or the "clicky here to update", (not currently displayed).

I'm hoping to migrate to a new box soon as this old Thinkpad is showing signs of warning (old, W540..) So, unless you feel I should force the change to the 128 stream, I'll hold here in ESR autoupdate.

Sorry for the drift. I know you're chasing a bug here.
